binzx/otomi

```diff
@@ -288,7 +289,7 @@ if [[ $calling_args == *'-non-interactive'* ]] ||
   it=''
 fi
 
-network='--network host'
+network=${CUSTOM_NETWORK:-'--network host'}
```

Comment:
It is still not clear to me why otomi container requires host network.

Reply:
load balancer?

Reply:
This discussion is not related to this PR, so I will resolve this conversation.